lib/message_sender.rb

```diff
@@ -0,0 +1,17 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+ class Message_sender
```

Comment:
Encapsulating the Twilio communications in a separate class is an excellent design decision.  However, watch your naming - this class should be `MessageSender` not `Message_sender`.  Also note that `attr_accessor :message` has no meaning within the current context of this class.